File: 800/282A_Bits++.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int n, contador = 0; cin>>n;
    
    while(n--){
        string operador; cin>>operador;
        if((operador == "++X")  || (operador == "X++")){
            contador++;
        }
        if((operador == "--X") || (operador == "X--")){
            contador--;
        }
    }

    cout<<contador;
    return 0;
}

File: 800/282A_Bits++.py
n = int(input())
contador = 0

for _ in range(n):
    operador = input().strip()   
    if (operador == "++X") or (operador == "X++"):
        contador += 1

    if (operador == "--X") or (operador == "X--"):
        contador -= 1

print(contador)
